docs/framework/guides/contributing/git-commit-message-convention.md

```diff
@@ -47,23 +47,33 @@ Each commit can contain additional notes which will be inserted into the changel
 | Type | Is backward compatible? |
 | --- | --- |
 | `NOTE` | Yes |
-| `BREAKING CHANGES` or `BREAKING CHANGE` | No |
+| `MAJOR BREAKING CHANGE`, `MINOR BREAKING CHANGE` or `BREAKING CHANGE` | No |
 
-If any visible change contains the `BREAKING CHANGE` note, the next release will be marked as `major` automatically.
+The `BREAKING CHANGE` is an alias to a `MAJOR BREAKING CHANGE`.
+
+If any change contains the `MAJOR BREAKING CHANGE` note, the next release will be marked as `major` automatically.
+
+For reference on how to identify minor or major breaking change see the {@link framework/guides/support/versioning-policy versioning policy guide}.
+
+### Package name
+
+Most commits are related to one or more packages. Each affected package should be listed in parenthesis following the commit type. A package that was the most impacted by the change should be listed first.
+
+It is, however, possible to skip this part if five or more packages are affected.
```

Comment:
I wouldn't specify the exact number but rather explain the rationale behind this. The point is that if a change applies to more than 4-5 packages, then there's a great change that it's a quite generic change it's better described unscoped.

Also, the entire section is actually incorrect because a single changelog item may apply to just one package. The point is that a single message can contain multiple items. See https://github.com/ckeditor/ckeditor5/issues/6058#issuecomment-611482923.

Reply:
Handled in https://github.com/ckeditor/ckeditor5/pull/6711#pullrequestreview-404126213.